metric_learn/base_metric.py

```diff
@@ -333,7 +332,7 @@ def predict(self, pairs):
       The predicted learned metric value between samples in every pair.
     """
     check_is_fitted(self, ['threshold_', 'transformer_'])
-    return 2 * (self.decision_function(pairs) > - self.threshold_) - 1
+    return 2 * (self.decision_function(pairs) >= - self.threshold_) - 1
```

Comment:
This is the rule used in scikit-learn for thresholds, see for instance here: https://github.com/scikit-learn/scikit-learn/blob/62d205980446a1abc1065f4332fd74eee57fcf73/sklearn/metrics/ranking.py#L377-L381

Reply:
Maybe `- self.decision_function(pairs) <= self.threshold_` is more intuitive?

Reply:
I agree, done